packages/specs/src/runtime/index.ts

```diff
@@ -0,0 +1,42 @@
+import { discovery } from './discovery';
+import { babe } from './babe';
+import { metadata } from './metadata';
+import { RuntimeApisSpec, RuntimeCallSpec, RuntimeApiSpec } from '@delightfuldot/types';
+import { blake2AsHex } from '@polkadot/util-crypto';
+import { stringSnakeCase } from '@delightfuldot/utils';
+
+export const runtimeApisModules: RuntimeApisSpec = { discovery, babe, metadata };
+
+export const runtimeApisNames = Object.values(runtimeApisModules)
+  .map((one) => Object.keys(one))
+  .flat();
+
+export const runtimesApisSpecs = Object.keys(runtimeApisModules)
+  .map((module) => {
+    return Object.keys(runtimeApisModules[module]).map((runtime) => {
+      return runtimeApisModules[module][runtime].map(
+        (spec) => ({ ...spec, moduleName: module, runtimeApiName: runtime }) as RuntimeApiSpec,
+      );
+    });
+  })
+  .flat(2);
+
+export const runtimeCallsSpecs = runtimesApisSpecs
+  .map(({ methods, runtimeApiName, version }) => {
+    return Object.keys(methods).map(
+      (methodName) => ({ ...methods[methodName], methodName, runtimeApiName, version }) as RuntimeCallSpec,
+    );
+  })
+  .flat();
```

Comment:
We should rename this and explicitly add type for it as well:
```suggestion
export const runtimeApisSpec: RuntimeApisSpec = { discovery, babe, metadata };

export const runtimeApiNames: string[] = Object.values(runtimeApisSpec)
  .map((one) => Object.keys(one))
  .flat();

export const runtimesApiSpecs: RuntimeApiSpec[] = Object.keys(runtimeApisSpec)
  .map((moduleName) => {
    return Object.keys(runtimeApisSpec[moduleName]).map((runtimeApiName) => {
      return runtimeApisSpec[moduleName][runtimeApiName].map(
        (spec) => ({ ...spec, moduleName, runtimeApiName }) as RuntimeApiSpec,
      );
    });
  })
  .flat(2);

export const runtimeCallSpecs: RuntimeCallSpec[] = runtimesApiSpecs
  .map(({ methods, runtimeApiName, version }) => {
    return Object.keys(methods).map(
      (methodName) => ({ ...methods[methodName], methodName, runtimeApiName, version }) as RuntimeCallSpec,
    );
  })
  .flat();
```